ee-feature-pack/common/pom.xml

```diff
@@ -51,6 +51,13 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <dependency>
+                <groupId>${ee.maven.groupId}</groupId>
+                <artifactId>wildfly-standard-test-bom</artifactId>
+                <version>${ee.maven.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
```

Comment:
I don't think should be necessary. There's no testing involved with this module.

Reply:
Right, I have updated the code.
Thanks